src/app/template.tsx

```diff
@@ -10,51 +10,87 @@ import {
   UserMenu,
   SideNavigation,
 } from "@/components/global";
+import { ContentStackContextProvider } from "@/contexts/ContentStackContext";
+import { ComponentFields, ContentPageGroup } from "@/utils/ContentStack/types";
+import {
+  getComponents,
+  getContentPageGroups,
+} from "@/utils/ContentStack/getContentstackResources";
```

Comment:
path change

Reply:
what should this change to? are we trying to kebab case subdirectories?